test/component-bind.spec.ts

```diff
@@ -359,24 +319,6 @@ describe('component', () => {
       value: string | number;
     }
 
-    it('throws if target value has changed w/o event to value rejected by type guard', () => {
-      let value: number;
-      class CustomChild extends Composite {
-        set bar(v: number) { value = v; }
-        get bar() { return value; }
-      }
-      const guarded = new ComponentWithTypeGuard();
-      const child = new CustomChild({id: 'foo'});
-      guarded.append(child);
-
-      child.bar = 12;
-
-      expect(() => guarded.value).to.throw(
-        'Binding "value" <-> "#foo.bar" failed to provide ComponentWithTypeGuard property "value": '
-        + 'Type guard rejected value "12".'
-      );
```

Comment:
Didn't investigate thoroughly why those tests were generally removed, did the type safety decrease?

Reply:
No, it increased. This check now happens automatically, not when calling the getter.